### ISSUE TITLE
./gradlew docs fails

### ISSUE THREAD

**Issue by jameskleeh** [micronaut-projects/micronaut-build#174]
Not seeing this in every repo, but found in `micronaut-rabbitmq` and `micronaut-kotlin`

This is the workaround that was added to resolve the issue

https://github.com/micronaut-projects/micronaut-kotlin/commit/ed158896305fd304e87a6f077b76cf33566e875e

**Comment by ojebari-ma** [comment]
Can you please confirm that this is still an issue in the latest version of Micronaut?

**Comment by alvarosanchez** [comment]
Thanks for the report. I am closing this because the issue has been waiting on reporter confirmation for more than 30 days.

The original report from 2021 described `./gradlew docs` failing in `micronaut-rabbitmq` and `micronaut-kotlin`, and linked the workaround commit `micronaut-projects/micronaut-kotlin@ed158896305fd304e87a6f077b76cf33566e875e`. The follow-up discussion did not identify a confirmed current failure in `micronaut-build`: on 2021-10-28 maintainers noted the Gradle warning had been seen but was not fixed then, and on 2025-05-25 we asked whether this still reproduces on the latest Micronaut version. No reproduction details, affected current versions, or confirmation have been provided since that 2025-05-25 clarification request.

With the issue still labeled `status: awaiting feedback` and no response after the clarification window, there is not enough current evidence for QA to route implementation work safely. If this still reproduces, please open a new issue with the Micronaut Build version, the affected consuming repository, the exact `./gradlew docs` command, the full failure output, and whether the old Kotlin workaround still changes the result.

---
###### ✨ This message was AI-generated using gpt-5.5
